doc/guides/cross-network-inet-multicast-howto.md

```diff
@@ -34,7 +34,7 @@ While it is not particularly important where you clone and build the _mcproxy_ d
 % make
 ```
 
-## Step 4: Establish the Happy Topology
+## Step 4: Establish the happy topology
```

Comment:
`Happy` is a proper name and should be capitalized

Reply:
or if it isnt, we should change it to lowercase below

Reply:
No, you're right, should be capitalized here.  Should only be lowercase when we're talking about the repo or command. Fixed.